src/Microsoft.Azure.SignalR/HubHost/ServiceLifetimeManagerBase.cs

```diff
@@ -267,6 +288,86 @@ public override Task RemoveFromGroupAsync(string connectionId, string groupName,
             return WriteAckableMessageAsync(message, cancellationToken);
         }
 
+#if NET7_0_OR_GREATER
+        public override async Task<T> InvokeConnectionAsync<T>(string connectionId, string methodName, object[] args, CancellationToken cancellationToken = default)
```

Comment:
since it is in base, does it mean Management API also supports ClientInvocation? @JialinXin ? Shall we throw for Management API?